cli/run_top_counties_dataset.py

```diff
@@ -10,9 +10,9 @@
 
 @click.command("deploy-top-counties")
 @click.option(
-    "--run_validation",
-    "-r",
-    default=True,
+    "--disable_validation",
```

Comment:
most of the others are `--disable-validation` (click converts the name to th underscore in the function calls)

Reply:
coool done